verification.py

```diff
@@ -186,6 +196,16 @@ def _run_ms_coalescent_stats(self, args):
             df = pd.read_table(f)
         return df
 
+    def _run_discoal_coalescent_stats(self, args):
+        executable = ["./data/ms_summary_stats"]
```

Comment:
This won't work for discoal I think. The coalescent stats here are basically internal counts from ms --- the number of RE and CA events. The ms_summary_stats executable here is a patched version of ms that spits these out. Unless you want to do the same for discoal, we won't be able to do these comparisons.

Reply:
ah ok. will jettison these then.